src/components/ModelingSidebar/ModelingSidebar.tsx

```diff
@@ -24,6 +24,10 @@ interface BadgeInfoComputed {
   onClick?: MouseEventHandler<any>
 }
 
+function getPlatformString(): 'web' | 'desktop' {
+  return isDesktop() ? 'desktop' : 'web'
+}
```

Comment:
Hmm, we already use "platform" to mean "OS" (see `src/lib/utils.ts`). Ideally we don't use the same term to talk about multiple things. I think I like this new definition better. We should probably rename the existing uses at some point to `osFamily` or something.